perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -222,28 +222,15 @@ sub pay_cancel : Local : Args(2) {
     my $property_id = $c->get_param('property_id');
 
     $c->forward('check_payment_redirect_id', [ $id, $token ]);
-    my $p = $c->stash->{report};
 
     $c->forward('/auth/get_csrf_token');
 
-    # XXX Photos not present in report as yet
-    my $saved_data = {
-        "chosen_date" => $p->get_extra_field_value('DATE'),
-        "location" => $p->get_extra_field_value('CREW NOTES'),
-        #"location_photo_fileid" => '???',
-        "name" => $p->name,
-        "email" => $p->user->email,
-        "phone" => $p->user->phone,
-        "resident" => "Yes",
-    };
-    my @fields = grep { $_->{name} =~ /ITEM_/ } @{$p->get_extra_fields};
-    foreach (@fields) {
-        my ($id) = $_->{name} =~ /ITEM_(\d+)/;
-        $saved_data->{"item_" . ($id+0)} = {
-            item => $_->{value},
-            #photo_fileid => '???',
-        };
-    }
+    my $p = $c->stash->{report};
+    my $saved_data = $c->cobrand->waste_reconstruct_bulky_data($p);
+    $saved_data->{name} = $p->name;
+    $saved_data->{email} = $p->user->email;
+    $saved_data->{phone} = $p->user->phone;
+    $saved_data->{resident} = 'Yes';
```

Comment:
Just curious why these 4 didn't go into `waste_reconstruct_bulky_data`?

Reply:
(oh unless it's because there's nothing cobrand-specific about them?)

Reply:
Yeah, thought might as well split it like that.